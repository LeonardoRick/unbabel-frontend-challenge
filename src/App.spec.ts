import AppVue from '@/App.vue';
import { mount } from '@vue/test-utils';
import { createPinia } from 'pinia';
import router from '@/router';

describe('TranscriptionItem', () => {
    it('should mount', () => {
        expect(
            mount(AppVue, {
                global: {
                    plugins: [createPinia(), router]
                }
            }).html()
        ).toMatchSnapshot();
    });
});
