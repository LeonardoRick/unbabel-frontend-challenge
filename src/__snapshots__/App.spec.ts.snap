// Vitest Snapshot v1

exports[`TranscriptionItem > should mount 1`] = `
"<header data-v-7a7a37b1=\\"\\">
  <div class=\\"inner-container\\">
    <h1><a href=\\"/home\\" class=\\"\\">Transcriptions</a></h1>
    <nav><a href=\\"/another-page\\" class=\\"about\\">About</a><button type=\\"button\\" class=\\"reset-button-style\\"><img src=\\"/src/assets/images/upload.svg\\" alt=\\"upload list of transcriptions\\"></button><button type=\\"button\\" class=\\"reset-button-style\\"><img src=\\"/src/assets/images/fetch-document.svg\\" alt=\\"fetch transcriptions\\"></button></nav>
  </div>
</header>
<main data-v-7a7a37b1=\\"\\">
  <!---->
</main>"
`;
